website/snippets/_sl-partner-links.md

```diff
@@ -11,7 +11,7 @@ The following tools integrate with the dbt Semantic Layer:
     icon="tableau-software"/>
 
   <Card
-    title="Google Sheets (Preview)"
+    title="Google Sheets"
```

Comment:
removed the 'preview' mention from the main avail integrations card parge too @rpourzand 

![Screenshot 2024-04-29 at 17 05 31](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/13da5d29-3fd8-420a-b629-ac7b936a224a)